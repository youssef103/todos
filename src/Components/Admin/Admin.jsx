import React from "react";

export const Admin = props => {
  return (
    <div>
      <p>Admin</p>
    </div>
  );
};
