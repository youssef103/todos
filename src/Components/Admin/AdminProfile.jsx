import React from "react";

export const AdminProfile = props => {
  return <div>AdminProfile</div>;
};
