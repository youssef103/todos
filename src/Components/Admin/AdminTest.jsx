import React from "react";

export const AdminTest = () => {
  return <div>AdminTest</div>;
};
