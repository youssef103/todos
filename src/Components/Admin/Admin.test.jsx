import { Admin } from "./Admin";

test("Should render Admin", () => {
  const wrapper = shallow(<Admin />);
  expect(wrapper).toMatchSnapshot();
});
