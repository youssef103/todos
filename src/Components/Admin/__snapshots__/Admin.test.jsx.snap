// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Admin 1`] = `
<div>
  Admin
</div>
`;
